docs/tutorials/porch-development-environment/README.md

```diff
@@ -0,0 +1,149 @@
+# Setting up a development environment for Porch
+
+This tutorial gives short instructions on how to set up a development environment for Porch. It outlines the steps to get a [kind](https://kind.sigs.k8s.io/) cluster up
+and running to which a Porch instance running in Visual Studio Code can connect to and interact with.
+
+# Setup kind with MetalLB and Gitea
+
+Follow steps 1-5 inclusive of the [Starting with Porch](https://github.com/nephio-project/porch/tree/main/docs/tutorials/starting-with-porch) tutorial. You now have two Kind clusters `management` and `edge1` running with Gitea installed on the `management` cluster. Gitea has the repositories `management` and `edge1` defined.
+
+> **_NOTE:_** This [setup script](bin/setup.sh) automates steps 1-5 of the Starting with Porch tutorial. You may need to adaot this script to your local environment.
+
+> **_NOTE:_** This [cleardown script script](bin/cleardown.sh) clears everything down by deleting the `management` and `edge1` Kind clusters. USE WITH CARE.
+
+
+You can reach the Gitea web interface on the address reported by the following command:
+```
+kubectl get svc -n gitea gitea        
```

Comment:
The script seems to set the default context to the edge cluster, so may need to switch to the management context.
kubectl config use-context kind-management

Reply:
Done, added command to the end of the script